radio/src/targets/common/arm/stm32/adc_driver.cpp

```diff
@@ -24,10 +24,12 @@
   // not needed
 #elif defined(PCBX10)
   const int8_t ana_direction[NUM_ANALOGS] = {1,-1,1,-1,  -1,-1,-1, -1,1,1,1,  -1};
-#elif defined(PCBX9E) && defined(HORUS_STICKS)
-  const int8_t ana_direction[NUM_ANALOGS] = {1,-1,1,-1,  -1,-1,-1,1, -1,1,1,1,  -1};
 #elif defined(PCBX9E)
-  const int8_t ana_direction[NUM_ANALOGS] = {1,1,-1,-1,  -1,-1,-1,1, -1,1,1,1,  -1};
+  #if defined(HORUS_STICKS)
```

Comment:
Yes when the code is all `#` preproc stuff then indenting works nicely I think.  When mixed with C code I end up not using indents.  But when the block spans more than a few lines or has nested blocks I find it important to comment all the `#else`s and `#endif`s copiously.   

Of course the best way to view it in any case is when the IDE does proper highlighting of "dead" code.